js/script.js

```diff
@@ -0,0 +1,58 @@
+let command;
+
+while (command !== "1" || command !== "2" || command !== "3" || command !== "4") {
+
+    command = prompt(`Calculator
```

Comment:
The program finishes only if I type `1, 2, 3, 4`

If another symbol has been entered it is continuing suggestions to enter data, even if I am clicking  `cancel` button on the `prompt` window

Let's don't show the next `prompt` window in case the user click the `cancel` button on the `prompt`